### DISCUSSION TITLE
Invitation to Collaborate 🚀

### DISCUSSION THREAD

**Discussion by vn7n24fzkq** [vn7n24fzkq/github-profile-summary-cards#170] [Show and tell]
Currently, this repository is hosted under my personal account, and I am eager to have it moved to an organization, fostering increased collaboration in the project.

I am personally covering the expenses on Vercel. If you are interested in contributing to the development or joining the organization, please feel free to contact me via email or comment below.
Your response will be greatly appreciated.

**Comment by tobidelly** [comment]
Would love to collaborate, what can I do to help it become better?
